compositions/cluster-k8s/block-sync/historical/64-square-size/1-3-32-ipld.toml

```diff
@@ -0,0 +1,114 @@
+[metadata]
+  name = "blocksync-historical"
+  author = "derrandz"
+
+[global]
+  plan = "celestia"
+  case = "blocksync-historical"
+  total_instances = 36
+  builder = "docker:generic"
+  runner = "cluster:k8s"
+  disable_metrics = false
+
+# we define 1 validator that produces 500kb blocks
+# which results in eds of size 64
+[global.run.test_params]
+  execution-time = "20"
+  persistent-peers = "1"
+  submit-times = "10"
+  msg-size = "500000"
+  validator = "1"
+  bridge = "3"
+  full = "16"
+  historical = "16"
+  block-height = "30"
+  otel-collector-address = "178.128.163.171:4318"
```

Comment:
Can you please give a comment what is this IP? 
I suppose it's a DO one?

Reply:
yes, removed it. that was my instance, replaced with an informative comment